### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][Eclipse][Wipro]:Unable to resize the columns in " Service Principal Creation Status "window for Azure signin automatically

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1529]
1. Open and Start Eclipse.
2. Open your project with Eclipse.
3. Press "Alt+T" to open  Tools menu, Navigate to  "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Automated" radio button, and navigate to "New" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. After sign In Create Authentication Files window will opened .
7. In "Create Authentication files window" navigate and select the "subscriptions" that you want to use, choose your "destination directory", and then select Start.
8. In the Service Principal Creation Status dialog , navigate to results and try to resize the column's

**Expected Result:**
Keyboard users should able to do resize by using keyboard

**Actual Result:**
Keyboard user's unable to do resize the columns by using keyboard 

![unable to resize columns by using keyboard in create service principle creation status window](https://user-images.githubusercontent.com/22956130/35679993-f5f151fc-077e-11e8-984a-5b96b97a6b65.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by SummerSun** [comment]
Same as #1368

**Comment by Eskibear** [comment]
external issue: 
Ref:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=308107

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749964
